reference/3.0/Microsoft.PowerShell.Core/About/about_Regular_Expressions.md

```diff
@@ -16,50 +16,132 @@ Describes regular expressions in Windows PowerShell.
+         as Ll, Nd, Z, IsGreek, and IsBoxDrawing.
+Example: "abcd defg" -match "\p{Ll}+"
+
+Format:  \P{name}
+Logic:   Matches text not included in the groups and block ranges specified
+         in {name}.
+Example: 1234 -match "\P{Ll}+"
+
+Format:  \w
+Logic:   Matches any word character. Equivalent to the Unicode character
+         categories [\p{Ll}\p{Lu}\p{Lt}\p{Lo}\p{Nd}\p{Pc}]. If ECMAScript-
+         compliant behavior is specified with the ECMAScript option, \w is
+         equivalent to [a-zA-Z_0-9].
+Example: "abcd defg" -match "\w+" (this matches abcd)
+
+Format:  \W
+Logic:   Matches any nonword character. Equivalent to the Unicode categories
+         [^\p{Ll}\p{Lu}\p{Lt}\p{Lo}\p{Nd}\p{Pc}].
+Example: "abcd defg" -match "\W+" (this matches the space)
+
+Format:  \s
+Logic:   Matches any white-space character. Equivalent to the Unicode
+         character categories [\f\n\r\t\v\x85\p{Z}].
+Example: "abcd defg" -match "\s+"
+
+Format:  \S
+Logic:   Matches any non-white-space character. Equivalent to the Unicode
+         character categories [^\f\n\r\t\v\x85\p{Z}].
+Example: "abcd defg" -match "\S+"
+
+Format:  \d
+Logic:   Matches any decimal digit. Equivalent to \p{Nd} for Unicode and
+         [0-9] for non-Unicode behavior.
+Example: 12345 -match "\d+"
+
+Format:  \D
+Logic:   Matches any nondigit. Equivalent to \P{Nd} for Unicode and [^0-9]
+         for non-Unicode behavior.
+Example: "abcd" -match "\D+"
+```
 Windows PowerShell supports the quantifiers available in .NET Framework
 regular expressions. The following are some examples of quantifiers.
 
-| Format | Logic | Example |
-| ------ | ----- | ------- |
-| *      | Specifies zero or more matches; for example, \wor (abc). Equivalent to {0,}. | "abc" -match "\w*" |
-| +      | Matches repeating instances of the preceding characters. | "xyxyxy" -match "xy+" |
-| ?      | Specifies zero or one matches; for example, \w? or (abc)?. Equivalent to {0,1}. | "abc" -match "\w?" |
-| {n}    | Specifies exactly n matches; for example, (pizza){2}. | "abc" -match "\w{2}" |
-| {n,}   | Specifies at least n matches; for example, (abc){2,}. | "abc" -match "\w{2,}" |
-| {n,m}  | Specifies at least n, but no more than m, matches. | "abc" -match "\w{2,3}" |
+```
+Format:  *
+Logic    Specifies zero or more matches; for example, \wor (abc). Equivalent
+         to {0,}.
+Example: "abc" -match "\w*"
+
+Format:  +
+Logic:   Matches repeating instances of the preceding characters.
+Example: "xyxyxy" -match "xy+"
+
+Format:  ?
+Logic:   Specifies zero or one matches; for example, \w? or (abc)?.
+         Equivalent to {0,1}.
+Example: "abc" -match "\w?"
+
+Format:  {n}
+Logic:   Specifies exactly n matches; for example, (pizza){2}.
+Example: "abc" -match "\w{2}"
+
+Format:  {n,}
+Logic:   Specifies at least n matches; for example, (abc){2,}.
+Example: "abc" -match "\w{2,}"
+
+Format:  {n,m}
+Logic:   Specifies at least n, but no more than m, matches.
+Example: "abc" -match "\w{2,3}"
+```
```

Comment:
This was a table and it was totally broken. It will not render properly with Pandoc for the .txt output. when we can use real markdown for these we can reformat it.